checkout-ui-custom/src/controller/DeliverController.js

```diff
@@ -155,6 +155,9 @@ const DeliverController = (() => {
 
     if (document.forms['bash--delivery-form']) {
       document.forms['bash--delivery-form'].reset();
+      // reset prepopulated lat and long
```

Comment:
What about previous uses of this address? I Is it the GPS set on the OrderForm or on the Address?

Reply:
It's a precaution for when user switches addresses, to ensure that the next address doesn't have the previous addresses coords.

Reply:
👍